### PR TITLE
[scitedotai/scite-zotero-plugin#19] Add Total Smart Citations / Total Citing Publication columns

### PR DESCRIPTION
**Pull request by u-ashish**
## Changelog

Add columns to expose the total count of Smart Citations and the total number of Citing Publications.

## Screenshots

Screen Shot 2021-06-10 at 10 01 13 AM

Screen Shot 2021-06-10 at 10 01 07 AM

### CODE REVIEW

File: content/zoteroPane.xul [hunk 1]
```diff
@@ -35,6 +35,22 @@
       zotero-persist="width ordinal hidden sortActive sortDirection"
     />
     <splitter class="tree-splitter"/>
+    <treecol
+      id="zotero-items-column-total"
+      label="Total Smart Citations"
```
Comment (domenicrosati) [inline content/zoteroPane.xul:40]: 👍

File: skin/default/overlay.css [hunk 2]
```diff
@@ -35,3 +35,17 @@ button *, button:focus *
   list-style-image: url(chrome://zotero-scite/skin/loading.jpg);
   height: 10px;
 }
+
+#zotero-items-tree treechildren::-moz-tree-image(scite-state-total)
+{
+  margin: 1px 5px 0;
+  list-style-image: url(chrome://zotero-scite/skin/total.png);
```
[the remark this answers is not in the document]
Comment (u-ashish) [reply skin/default/overlay.css:42]: Oh ya, do we have a pic for total lol

**Review by domenicrosati — APPROVED** [review]
lgtm + question about total.png

File: skin/default/overlay.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (u-ashish) [reply skin/default/overlay.css:42]: Just gonna remove this line! We don't have that asset anyway.

**Comment by u-ashish** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 1.0.7.100](https://github.com/scitedotai/scite-zotero-plugin/releases/download/builds/zotero-scite-1.0.7.100.xpi) ("Merge pull request #20 from scitedotai/additional-tally-columns\n\n[scitedotai/scite-zotero-plugin#19] Add Total Smart Citations / Total Citing Publication columns")

Install in Zotero by downloading [test build 1.0.7.100](https://github.com/scitedotai/scite-zotero-plugin/releases/download/builds/zotero-scite-1.0.7.100.xpi), opening the Zotero "Tools" menu, selecting "Add-ons", open the gear menu in the top right, and select "Install Add-on From File...".

**Pull request merged**